src/components/Navbar.jsx

```diff
@@ -0,0 +1,24 @@
+import { Link } from "react-router-dom"
+
+const Navbar = () => {
+    return(
+        <nav className="navbar" id="navbar">
+            <ol>
+                <li><a href="/">Home</a></li>
+                <li>
+                    <div className="dropdown">
+                    <Link to = '/articles' className="dropbtn">Articles</Link>
+                    <div className="dropdown-content">
+                        <Link className="dropdown-links" to ='/articles/coding'>Coding</Link>
+                        <Link className="dropdown-links" to ='/articles/football'>Football</Link>
+                        <Link className="dropdown-links" to ='/articles/cooking'>Cooking</Link>
```

Comment:
Hard coded topics is fine at this point, but to make sure it's always up to date, better to make the api call and generate the topics by using the topic array comes back from the api